src/runtime/cmd/kata-runtime/kata-iptables.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	containerdshim "github.com/kata-containers/kata-containers/src/runtime/pkg/containerd-shim-v2"
+	"github.com/kata-containers/kata-containers/src/runtime/pkg/katautils"
+	"github.com/kata-containers/kata-containers/src/runtime/pkg/utils/shimclient"
+	"github.com/urfave/cli"
+)
+
+var (
+	sandboxID    string
+	iptablesFile string
```

Comment:
delete this, `iptablesFile` is not being used.

Reply:
Actually, we just use it for testing as part of the kata-runtime CLI